packages/components/src/GuidanceBlock/_docs/GuidanceBlock.stickersheet.stories.tsx

```diff
@@ -0,0 +1,120 @@
+import React from "react"
+import { Meta } from "@storybook/react"
+import { Heading } from "~components/Heading"
+import { EmptyStatesPositive, Informative } from "~components/Illustration"
+import { Tag } from "~components/Tag"
+import { Text } from "~components/Text"
+import {
+  StickerSheet,
+  StickerSheetStory,
+} from "~storybook/components/StickerSheet"
+import { GuidanceBlock, GuidanceBlockProps } from "../index"
+import { variantsMap } from "../types"
+
+export default {
+  title: "Components/GuidanceBlock",
+  parameters: {
+    chromatic: { disable: false },
+    controls: { disable: true },
+  },
+} satisfies Meta
+
+const GENERIC_PROPS = {
+  persistent: true,
+  illustration: <Informative alt="" />,
+  actions: {
+    primary: {
+      label: "Action",
+      onClick: () => {
+        alert("tada: 🎉")
+      },
+    },
+    secondary: {
+      label: "Pass",
+      onClick: () => {
+        alert("tada: 🎉")
+      },
+    },
+  },
+}
+
+const TEXT_PROPS: GuidanceBlockProps = {
+  ...GENERIC_PROPS,
+  text: {
+    title: "This is the Guidance block title",
+    description:
+      "Mussum Ipsum, cacilds vidis litro abertis. Suco de cevadiss, é um leite divinis, " +
+      "qui tem lupuliz, matis, aguis e fermentis. Mé faiz elementum girarzis, nisi eros vermeio.",
+  },
+}
+
+const CONTENT_PROPS: GuidanceBlockProps = {
+  ...GENERIC_PROPS,
+  content: (
+    <>
+      <Tag>Early Access</Tag>
+      <Heading variant="heading-3">{TEXT_PROPS.text.title}</Heading>
+      <Text variant="body">{TEXT_PROPS.text.description}</Text>
+    </>
+  ),
+}
+
+const StickerSheetTemplate: StickerSheetStory = {
```

Comment:
Should we also have the stacked variants sent to chromatic for safety since there a visual difference in the layout? either that or send the individual stories